src/pyscipopt/scip.pxi

```diff
@@ -2121,108 +3766,313 @@ cdef class Model:
-        return SCIPgetRowActivity(self._scip, row.scip_row)
+    def getRowActivity(self, Row row):
+        """
+        Returns the activity of a row in the last LP or pseudo solution.
+
+        Parameters
+        ----------
+        row : Row
+
+        Returns
+        -------
+        float
+
+        """
+        return SCIPgetRowActivity(self._scip, row.scip_row)
+
+    def getRowLPActivity(self, Row row):
+        """
+        Returns the activity of a row in the last LP solution.
+
+        Parameters
+        ----------
+        row : Row
+
+        Returns
+        -------
+        float
 
-    def getRowLPActivity(self, Row row):
-        """returns the activity of a row in the last LP solution"""
+        """
         return SCIPgetRowLPActivity(self._scip, row.scip_row)
 
     # TODO: do we need this? (also do we need release var??)
     def releaseRow(self, Row row not None):
-        """decreases usage counter of LP row, and frees memory if necessary"""
+        """
+        Decreases usage counter of LP row, and frees memory if necessary.
+
+        Parameters
+        ----------
+        row : Row
+
+        """
         PY_SCIP_CALL(SCIPreleaseRow(self._scip, &row.scip_row))
 
     def cacheRowExtensions(self, Row row not None):
-        """informs row, that all subsequent additions of variables to the row should be cached and not directly applied;
+        """
+        Informs row, that all subsequent additions of variables to the row
+        should be cached and not directly applied;
         after all additions were applied, flushRowExtensions() must be called;
-        while the caching of row extensions is activated, information methods of the row give invalid results;
-        caching should be used, if a row is build with addVarToRow() calls variable by variable to increase the performance"""
+        while the caching of row extensions is activated, information methods of the
+        row give invalid results; caching should be used, if a row is build with addVarToRow()
+        calls variable by variable to increase the performance.
+
+        Parameters
+        ----------
+        row : Row
+
+        """
         PY_SCIP_CALL(SCIPcacheRowExtensions(self._scip, row.scip_row))
 
     def flushRowExtensions(self, Row row not None):
-        """flushes all cached row extensions after a call of cacheRowExtensions() and merges coefficients with equal columns into a single coefficient"""
+        """
+        Flushes all cached row extensions after a call of cacheRowExtensions()
+        and merges coefficients with equal columns into a single coefficient
+
+        Parameters
+        ----------
+        row : Row
+
+        """
         PY_SCIP_CALL(SCIPflushRowExtensions(self._scip, row.scip_row))
 
     def addVarToRow(self, Row row not None, Variable var not None, value):
-        """resolves variable to columns and adds them with the coefficient to the row"""
```

Comment:
stopped here

Reply:
sorry for the intrusion 😄 but since you'll still continue reviewing that might be helpful. For one liner suggestions like these you can add them directly from the code review and the PR author can just accept them and a commit will be created.

Reply:
Oh no, intrude away! Thanks Mo, and sorry Mark!

Reply:
No problem!